mod-linked-data/src/main/resources/citation/mod-linked-data/features/suppress-flags/suppress-flags.feature

```diff
@@ -30,6 +30,6 @@ Feature: Suppress Flags in Inventory
     # Search the Check Suppress Flags values
     Given path 'search/linked-data/works'
     And param query = query
-    And retry until response.content[0].instances[0].suppress.staff == true && response.content[0].instances[0].suppress.fromDiscovery == true
+    And retry until response.content[0] && response.content[0].instances[0] && response.content[0].instances[0].suppress && response.content[0].instances[0].suppress.staff == true && response.content[0].instances[0].suppress.fromDiscovery == true
```

Comment:
I am getting a failure in this line without these additional conditions.

Reply:
yeah, it's unstable for me too